security/index.adoc

```diff
@@ -0,0 +1,86 @@
+[id="security-compliance-overview"]
+= {product-title} security and compliance
+include::modules/common-attributes.adoc[]
+:context: security-compliance-overview
+
+toc::[]
+
+// TODO: Need some help with an intro blurb
+
+[id="security-overview"]
+== Security overview
+
+It is important to understand how to properly secure various aspects of your {product-title} cluster.
+
+[discrete]
+=== Container security
+
+A good starting point to understanding {product-title} security is to review the concepts in xref:../security/container_security/security-understanding.adoc#security-understanding[Understanding container security]. This and subsequent sections provide a high-level walkthrough of the container security measures available in {product-title}, including solutions for the host layer, the container and orchestration layer, and the build and application layer. These sections also include information on:
```

Comment:
IBM SG would suggest something like:

```
These sections also include information on the following topics:
```

to avoid continuing the lead-in sentence into a list. Also dropping the punctuation from all the list items since they're fragments.